cinema/models.py

```diff
@@ -111,17 +132,17 @@ def clean(self):
 
     def save(
         self,
-        force_insert=False,
-        force_update=False,
-        using=None,
-        update_fields=None,
-    ):
+        force_insert: bool = False,
+        force_update: bool = False,
+        using: Any = None,
+        update_fields: Any = None,
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
invalid typehint for using and update_fields